component-selenium-grid.yml

```diff
@@ -1,104 +1,264 @@
+    input.windows-user-password: "qwerty12!"
+    input.selenium-version: "2.47.0"
+    input.ie-driver-version: "2.47.0"
+    input.chrome-driver-version: "2.16"
+    #conf.recipes: "https://s3.amazonaws.com/qubell-starter-kit-artifacts/qubell-bazaar/component-selenium-grid-cookbooks-stable-9292b20.tar.gz"
+    input.recipes: "https://dl.dropboxusercontent.com/u/250836/selenium.tar.gz"
+    input.java-win-url: "https://s3.amazonaws.com/qubell-starter-kit-artifacts/deps/jdk-7u71-windows-x64.exe"
+    input.mirror: "http://adp-repo.s3-website-us-east-1.amazonaws.com/cygwin"
+    input.packages: "openssh,curl,util-linux,procps,wget,unzip"
   interfaces:
-    conf: 
-      selenium-version: bind(selenium-grid#input.selenium-version)
-      slave-node-count: bind(selenium-grid#input.node-count)
-      recipes: bind(selenium-grid#input.recipes)
+    input:
+      hub-image: bind(selenium-hub#input.hub-image)
+      selenium-version: bind(selenium-hub#input.selenium-version, linux-nodes.selenium-linux-node#input.selenium-version, windows-nodes.selenium-windows-node#input.selenium-version)
+      ie-driver-version: bind(selenium-hub#input.ie-driver-version, linux-nodes.selenium-linux-node#input.ie-driver-version, windows-nodes.selenium-windows-node#input.ie-driver-version)
+      chrome-driver-version: bind(selenium-hub#input.chrome-driver-version, linux-nodes.selenium-linux-node#input.chrome-driver-version, windows-nodes.selenium-windows-node#input.chrome-driver-version)
+      windows-user-password: bind(selenium-hub#input.windows-user-password, windows-nodes.selenium-windows-node#input.windows-user-password)
+      linux-nodes-image: bind(linux-nodes.selenium-linux-node#input.linux-nodes-image)
+      linux-nodes-quantity: bind(proxy#input.linux-nodes-quantity)
+      windows-nodes-image: bind(windows-nodes.selenium-windows-node#input.windows-nodes-image)
+      windows-nodes-quantity: bind(proxy#input.windows-nodes-quantity)
+      recipes: bind(selenium-hub#input.recipes, linux-nodes.selenium-linux-node#input.recipes, windows-nodes.selenium-windows-node#input.recipes)
+      java-win-url: bind(selenium-hub#input.java-win-url, windows-nodes.selenium-windows-node#input.java-win-url)
+      mirror: bind(selenium-hub#input.mirror, windows-nodes.selenium-windows-node#input.mirror)
+      packages: bind(selenium-hub#input.packages, windows-nodes.selenium-windows-node#input.packages)
+    app-meta:
+      application-pic: "bind(metadata#output.application-pic)"
     endpoints:
-     console-url: bind(selenium-grid#result.console-url)
-     remote-url: bind(selenium-grid#result.remote-url)
+     console-url: bind(selenium-hub#result.console-url)
+     remote-url: bind(selenium-hub#result.remote-url)
     hosts:
-      hub: bind(selenium-grid#result.hub-host)
-      nodes: bind(selenium-grid#result.node-hosts)
-    actions:
-      scale: bind(selenium-grid#actions.scale)
-      re-start: bind(selenium-grid#actions.re-start)
```

Comment:
selenium might have memory leaks, and it is recommended to restart cluster from time to time.